### PR TITLE
libusb-compat

### PR DESCRIPTION
**Pull request by hmaarrfk**
I want this for some old executable that hasn't updated to libusb1. 

I don't think code that old on windows and macosx used libusb0. 

Checklist

- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [ ] Source is from official source
- [ ] Package does not vend other packages
- [ ] Build number is 0
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/libusb-compat/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    - {{ pin_subpackage('libusb-compat') }}
+  # For now, start with linux builds ony
+  skip: True  # [not linux]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - libtool  # [unix]
+    - pkg-config  # [unix]
```
Comment (chrisburr) [inline recipes/libusb-compat/meta.yaml]:
```suggestion
    - pkg-config  # [unix]
    - make  # [unix]
```

File: recipes/libusb-compat/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  host:
```
Comment (chrisburr) [inline recipes/libusb-compat/meta.yaml]:
```suggestion
```

File: recipes/libusb-compat/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,45 @@
+{% set version = "0.1.12" %}
+
+package:
+  name: libusb-compat
+  version: {{ version }}
+
+source:
+  url: http://downloads.sourceforge.net/libusb/libusb-0.1.12.tar.gz
```
Comment (chrisburr) [inline recipes/libusb-compat/meta.yaml]:
```suggestion
  url: http://downloads.sourceforge.net/libusb/libusb-{{ version }}.tar.gz
```

File: recipes/libusb-compat/build.sh [hunk 4]
```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+# http://www.linuxfromscratch.org/blfs/view/6.3/general/libusb.html
+export CFLAGS="$CFLAGS -Wno-error=format-truncation"
+./configure --disable-build-docs --prefix=${PREFIX} --verbose
+make
+make install
```
Comment (bgruening) [inline recipes/libusb-compat/build.sh]: @hmaarrfk is there a make test or make check which we can run?

**Comment by hmaarrfk** [comment]
It seems make check fails with the same error as

https://github.com/libusb/libusb/issues/570#issuecomment-490420500

Which was shut down on the basis that the reporter was using version 0 of the library.

**Comment by hmaarrfk** [comment]
Thanks

**Pull request merged**